Alterando namespace

saulocalixto opened the pull request:
1 - Alterando namespaces para padrão ListElab.Projeto.
2 - Organizando usings, removendo os que não são usados, organizando-os por nome.

saulocalixto committed:
Alterando namespace

1 - Alterando namespaces para padrão ListElab.Projeto.
2 - Organizando usings, removendo os que não são usados, organizando-os por nome.

sampaiodias commented:
Creio que só existem mais dois tipos de problemas

1. Namespace dentro de outro namespace com o mesmo nome (ex.: ListElab.Servico.Servico)
2. Namespace com nome em inglês, sendo que o projeto é em português (ex.: ListElab.Dominio.CustomAttributes e ListElab.Contrato.Authentication)

saulocalixto committed:
Alterando namespaces

1 - NameSpace Servico.Servico passou a ser Servico.ServicosImplementados;
2 - Removendo nomes em inglês de namespaces.
